### ISSUE TITLE
And another yet crash in DBDelegate

### ISSUE THREAD

**Issue by NSStalker** [dropbox/dropbox-sdk-obj-c#338]
Hello there.

I use Your SDK 6.0.0 in my iOS app.

Here is the crash I found in FirebaseCrashlytics:

![Screenshot 2021-11-02 at 09 49 05](https://user-images.githubusercontent.com/5647878/139799216-ce61edf3-a0e1-4c6c-a699-ed18e20bba72.png)

**Comment by greg-db** [comment]
Thanks for reporting these! Do you happen to know the steps to reproduce this? Likewise, if you can share the relevant code/parameters that the app was running at the time of the crash, that may be helpful.

Also, the latest version of the SDK is v6.2.0, which contains a few bug fixes since v6.0.0, so please update to v6.2.0 and let us know if that seems to resolve any of these. Thanks!

**Comment by NSStalker** [comment]
Yes, I anticipated this question, but unfortunately, I don't know the exact steps either, and I can only guess. Also, I can't reproduce the crash. I suspect the app's module is waiting for changes to the DropBox’s folders using specific cursors.

Just in case I attached fragments of the code. The suspect module is:
KPDBSyncContentChangesObserver.

[waiting_for_db_changes.zip](https://github.com/dropbox/dropbox-sdk-obj-c/files/7462548/waiting_for_db_changes.zip)

----
**_Also, the latest version of the SDK is v6.2.0, which contains a few bug fixes since v6.0.0, so please update to v6.2.0 and let us know if that seems to resolve any of these._** 
Yes, I'm going to do this.
